VERSION.txt

```diff
@@ -1 +1 @@
-v0.7.2
+v0.8.0
```

Comment:
Is this file necessary? can we remove it?

Reply:
let's add a todo...

Reply:
It's referenced here for CLI versioning:
https://github.com/open-cluster-management-io/clusteradm/blob/bb97f150a2340e6cfe7d407c7a71af8c39eaeaf3/version.go#L9